GSA_Adapter/Convert/ToGsa/Loads/Load.cs

```diff
@@ -55,6 +62,18 @@ public static List<string> ToGsaString(this GravityLoad load, double[] unitFacto
             return forceStrings;
         }
 
+        /***************************************************/
+        /**** Fallback  Method                         ****/
+        /***************************************************/
+
+        private static List<string> ToGsaString(this ILoad load, double[] unitFactors)
+        {
+            Engine.Base.Compute.RecordError($"Pushing load of type {load.GetType()} is not implemented for GSA_Toolkit.");
```

Comment:
```suggestion
            NotSupportedWarning(load.GetType(), "Loads");
```

Can use the standardised warning message here.